scripts/convert.py

```diff
@@ -5,90 +5,49 @@
 import os
 
 
-def csv_to_json(dir_path, filename, output_dir_path=None, labels=[]):
-    """Converts CSV data file to JSON data file
-
-    dir_path -- path to directory containing CSV file
-    filename -- CSV file name
-    output_dir_path -- output directory where JSON file will be placed
+def csv_file_to_json(dir_path, filename, output_dir_path, labels):
+    """Converts CSV data file to JSON data file.
+
+    :param dir_path: path to directory containing CSV file.
+    :type dir_path: str
+    :param filename: CSV file name.
+    :type filename: str
+    :param output_dir_path: output directory where JSON file will be placed.
+    :type output_dir_path: str
+    :param labels: list of columns names.
+    :type labels: list[str]
     """
-    if output_dir_path is None:
-        return
+
+    labels = [] if labels is None else labels
```

Comment:
IMO you should choose one of two ways:
1. Use default argument `labels=None`, and then the line above.
OR
2. Leave arguments as is, and make sure, that method is called with `[]` instead of `None`, and then remove this line.

Also (If you stick with option 1) I would prefer 

```
if labels is None:
    labels = []
```

as it seems to be more readable.

Reply:
Ok, I'll stick to options 1 and will do the same thing with `output_dir_path`.